src/Data/Blob.hs

```diff
@@ -96,12 +95,10 @@ languageTagForBlobPair :: BlobPair -> [(String, String)]
 languageTagForBlobPair pair = showLanguage (languageForBlobPair pair)
   where showLanguage = pure . (,) "language" . show
 
--- JZ:TODO: I need to ask about what does this do.
 pathKeyForBlobPair :: BlobPair -> FilePath
-pathKeyForBlobPair = mergeEdit combine . bimap blobFpath blobFpath where
+pathKeyForBlobPair = mergeEdit combine . bimap blobFilePath blobFilePath where
```

Comment:
Can I know what does this return, as `a -> b` seems not to be a filepath?  

I do not dare not refactor this function as I am not sure about the return value

Reply:
`bimap blobFIlePath blobFilePath` gives a value of type `Edit FIlePath FilePath` (where `Edit` is from `Data.Edit`, which is then torn down with the `mergeEdit` function.

If you're in doubt, leave it for now: incremental changes about which we're sure are better than big changes about which we're not. (Hence why it's taken so long to get us to use typed paths!)